time stamp
uint16[6] angle # dynamixel: 12bit[ 0 ~ 4096]
uint8[6] temp # temperature
uint8[6] load # load
uint8[6] moving # moving
uint8[6] error # error
